zerver/lib/mention.py

```diff
@@ -302,6 +348,11 @@ def get_group_members(self, user_group_id: int) -> List[int]:
     def get_stream_name_map(self, stream_names: Set[str]) -> Dict[str, int]:
         return self.mention_backend.get_stream_name_map(stream_names)
 
+    def get_topic_info_map(
+        self, channel_topic: Set[ChannelTopicInfo]
+    ) -> Dict[ChannelTopicInfo, int]:
+        return self.mention_backend.get_topic_info_map(channel_topic)
```

Comment:
`channel_topic` is a bad variable name for a set of `ChannelTopicInfo` variable; it sounds like it's a single one :(.

Reply:
it's an object containing channel and topic names, so changed it to `channel_topic_names`